Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Generic/BoilerPlantEnabler.mo

```diff
@@ -0,0 +1,250 @@
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          lineThickness=0.1),
+        Rectangle(
+          extent={{-100,100},{100,-100}},
+          lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          lineThickness=5,
+          borderPattern=BorderPattern.Raised),
+        Text(
+          extent={{-120,146},{100,108}},
+          lineColor={0,0,255},
+          textString="%name"),
+        Ellipse(extent={{-80,80},{80,-80}}, lineColor={28,108,200},fillColor={170,255,213},
+          fillPattern=FillPattern.Solid),
+        Ellipse(extent={{-90,90},{90,-90}}, lineColor={28,108,200}),
+        Rectangle(extent={{-75,2},{75,-2}}, lineColor={28,108,200},
+          fillColor={28,108,200},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-66,46},{76,10}},
+          lineColor={28,108,200},
+          textString="START"),
+        Text(
+          extent={{-66,-8},{76,-44}},
+          lineColor={28,108,200},
+          textString="STOP")},
+          coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,100}})),
+  Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-160,-140},{160,140}})),
+  Documentation(info="<html>
+<p>
+Block that generates boiler plant enable signal according to ASHRAE RP-1711
+Advanced Sequences of Operation for HVAC Systems Phase II – Central Plants and
+Hydronic Systems (December 31, 2019), section 5.3.2.1, 5.3.2.2, and 5.3.2.3.
+</p>
+<p>
+The boiler plant should be enabled and disabled according to following sequences:
+</p>
+<ol>
+<li>
+An enabling schedule should be included to allow operators to lock out the 
+boiler plant during off-hour, e.g. to allow off-hour operation of HVAC systems
+except the boiler plant. The default schedule shall be 24/7 and be adjustable.
+</li>
+<li>
+The plant should be enabled in the lowest stage when the plant has been
+disabled for at least <code>boiPlaOffStaHolTimVal</code>, e.g. 15 minutes and: 
+<ul>
+<li>
+Number of boiler plant requests is greater than number of requests to be ignored,
+ie, <code>hotWatSupResReq</code> &gt; <code>nHotWatReqIgn</code>
+(<code>nHotWatReqIgn</code> should default to 0 and be adjustable), and,
+</li>
+<li>
+Measured outdoor air temperature is lower than boiler lockout temperature, ie,
+<code>TOut</code> &lt; <code>TLocOut</code>, and,
+</li>
+<li>
+The boiler enable schedule <code>boiEnaSchTab</code> is active.
+</li>
+</ul>
+</li>
+<li>
+The plant should be disabled when it has been enabled for at least 
+<code>boiPlaOnStaHolTimVal</code>, e.g. 15 minutes and:
+<ul>
+<li>
+Number of boiler plant requests is less than number of requests to be ignored,ie,
+<code>hotWatSupResReq</code> &le; <code>ignReq</code> for a time <code>staOnHeaWatReqTim</code>, or,
+</li>
+<li>
+Outdoor air temperature is 1 &deg;F greater than boiler lockout temperature,ie,
```

Comment:
1 degF should be replaced with the `<code>locDt</code>` after you introduce the parameter.